nixos/modules/module-list.nix

```diff
@@ -713,7 +713,6 @@
   ./services/networking/rxe.nix
   ./services/networking/sabnzbd.nix
   ./services/networking/searx.nix
-  ./services/networking/seeks.nix
```

Comment:
You need to use `mkRemovedOptionModule` `nixos/modules/rename.nix`

Reply:
Added in 46d2a3ba7f6a05576c64b78e7e70d5187cfd5db3